src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/Contracts/BackupConfigInfo.cs

```diff
@@ -0,0 +1,24 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+using System.Collections;
+using System.Collections.Generic;
+using Microsoft.SqlTools.ServiceLayer.Admin.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.DisasterRecovery.Contracts
+{
+    /// <summary>
+    /// Provides database info for backup.
+    /// </summary>
+    public class BackupConfigInfo: DatabaseInfo
+    {
+        public string RecoveryModel { get; set; }
+        public List<RestoreItemSource> LatestBackups { get; set; }
```

Comment:
>LatestBackups [](start = 39, length = 13)

is this collection the latest backups, or is it all the available backups?  If it's latest then it implies its not the full set of recovery item sources to me?  Is that correct understanding, otherwise, I'd rename something like RestoreItemSources or something like that.